### PR TITLE
Beta TS API Update Changes

### LINKED ISSUE #432: Switch to new TornStats api.
We should switch to [the new TornStats API](https://beta.tornstats.com/api) asap.

### PR DESCRIPTION
**Pull request by Sashank999**
Closes #432 .

### CODE REVIEW

**Comment by DeKleineKobini** [comment]
Updating your stats while not gaining any stats shows 'undefined'.

Screenshot

**Comment by Sashank999** [comment]
Fixed.
Gym _ TORN - Google Chrome 13-Mar-21 10_35_12 PM

File: torntools/scripts/content/gym/ttGym.js [hunk 1]
```diff
@@ -270,24 +271,21 @@ function displayGraph() {
...
-						if (result.deltaDefense !== 0) {
+						if (result.deltaDefense && result.deltaDefense !== 0) {
 							gains.push(`${numberWithCommas(result.deltaDefense, false)} Defense`);
 						}
-						if (result.deltaDexterity !== 0) {
+						if (result.deltaDexterity && result.deltaDexterity !== 0) {
 							gains.push(`${numberWithCommas(result.deltaDexterity, false)} Dexterity`);
 						}
-						if (result.deltaSpeed !== 0) {
+						if (result.deltaSpeed && result.deltaSpeed !== 0) {
```
Comment (DeKleineKobini) [inline torntools/scripts/content/gym/ttGym.js:283]: When `result.deltaSpeed` is 0, the `result.deltaSpeed` will already catch it (0 is falsely).

File: torntools/scripts/content/gym/ttGym.js [hunk 2]
```diff
@@ -270,24 +271,21 @@ function displayGraph() {
...
 							gains.push(`${numberWithCommas(result.deltaSpeed, false)} Speed`);
 						}
 
 						update_message += gains.join(", ") + ` since your last update ${result.age}.`;
 						if (gains.length === 0) update_message = `You have not gained any stats since your last update ${result.age}.`;
-
-						let info_div = doc.new({ type: "div", class: "tt-info-message", text: update_message });
-						graph_area.appendChild(info_div);
```
Comment (DeKleineKobini) [inline torntools/scripts/content/gym/ttGym.js:290]: Why remove this?

**Review by DeKleineKobini — CHANGES_REQUESTED** [review]
A message like `Your stats have been updated! You have gained 25,226 dexterity since your last manual update 1 minute ago. You have gained a total of 25,226 stats.` isn't really great imo (it's the TS API response).

File: torntools/scripts/content/gym/ttGym.js (hunk 1 above)
Comment (Sashank999) [reply torntools/scripts/content/gym/ttGym.js]: Changed.

File: torntools/scripts/content/gym/ttGym.js (hunk 2 above)
Comment (Sashank999) [reply torntools/scripts/content/gym/ttGym.js]: Changed.

**Pull request merged**